instance/tests/models/test_server.py

```diff
@@ -440,6 +440,14 @@ def test_status_transitions(self):
         self.assertEqual(server.status, ServerStatus.Ready)
```

Comment:
Please add tests for `Booting -> Unknown -> Booting`, and `Building -> Unknown -> Building` too.